src/libs/ajax.js

```diff
@@ -580,6 +580,11 @@ const Workspaces = signal => ({
       storageCostEstimate: async () => {
         const res = await fetchOrchestration(`api/workspaces/${namespace}/${name}/storageCostEstimate`, _.merge(authOpts(), { signal }))
         return res.json()
+      },
+
+      notifications: async () => {
+        const res = await fetchRawls(`notifications/workspace/${namespace}/${name}`, _.merge(authOpts(), { signal }))
```

Comment:
TOL: Do we actually need to make this call? There's only one entry, and it looks like the profile key is deterministic and could be derived from data we already have.

Reply:
@zarsky-broad had strong opinions about why we need to call this, I'll let him explain further

Reply:
My strong opinions™️ are based on those that I remember when we implemented this for FC, so I figured we'd match that for now. That was my default, I'm good being overridden.